exporter/redis.go

```diff
@@ -732,6 +737,12 @@ func (e *Exporter) scrapeRedisHost(scrapes chan<- scrapeResult, addr string, idx
 		}
 	}
 
+	if reply, err := c.Do("SLOWLOG", "LEN"); err == nil {
+		e.metricsMtx.RLock()
+		e.metrics["slowlog_len"].WithLabelValues(addr, e.redis.Aliases[idx], "slowlog len").Set(float64(reply.(int64)))
```

Comment:
You can remove `"slowlog len"` - the cmd label doesn't really add anything useful to the metric.

Reply:
Also, instead of `float64(reply.(int64)` can you please use [ParseFloat](https://golang.org/pkg/strconv/#ParseFloat) ? There's a couple of examples in the code already where that's being used.

Reply:
EngHeng~That's a good suggestoin~

Reply:
But the value whick the cmd 'slow len' returned is interface{},it is int64 actually.So i think that float64(reply.(int64) maybe is a better way.

Reply:
Ah yes, good catch, didn't see it's of type interface{} that can be cast to int64.